src/components/Layout/SearchBar.tsx

```diff
@@ -0,0 +1,109 @@
+import classNames from 'classnames';
+import { navigate } from 'gatsby';
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import { FaSearch } from 'react-icons/fa';
+
+import styles from './SearchBar.module.scss';
+
+interface ISearchBar {
+  className?: string;
+}
+
+const SearchBar: React.FC<ISearchBar> = (props) => {
+  const [
+    isMobileOpen,
+    setIsMobileOpen,
+  ] = useState(false);
+
+  const [
+    searchQuery,
+    setSearchQuery,
+  ] = useState('');
+
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const toggleMobileOpen = (): void => {
+    setIsMobileOpen(true);
+    setTimeout(() => {
+      inputRef.current?.focus();
+    }, 1);
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchQuery(event.target.value);
+  };
+
+  const handleSubmit = ((event: React.FormEvent) => {
+    event.preventDefault();
+    navigate(`/search-results/?q=${searchQuery}`);
```

Comment:
Let's remove the trailing `/` after `/search-results`. If left in, I think Google Analytics will differentiate `/search-results` and `/search-results/` as two different pages.

Reply:
This was added in because the production version of the bundle appended the `/`